packages/client-core/src/admin2/DefaultAdminRoutes.tsx

```diff
@@ -101,5 +103,12 @@ export const DefaultAdminRoutes: Record<string, AdminRouteStateType> = {
     component: Resources,
     access: false,
     icon: <HiOutlineTableCells />
+  },
+  routes: {
+    name: 'user:dashboard.routes',
+    scope: 'routes',
+    component: Routes,
+    access: false,
+    icon: <HiGlobeAlt />
```

Comment:
More accurate icon is: `HiOutlineGlobeAlt`

Reply:
changed!